vuepress/docs/v7.0/tutorials/solution/pda-tutorial.md

```diff
@@ -56,6 +56,42 @@ ent ecr deploy --repo="https://github.com/entando-samples/entando-process-driven
 
 ## Configuration
 
+::: warning
+---
+
+<details><summary>OpenShift 4.8 requires a workaround</summary>
+
+An incomplete role definition prevents Entando 7 from configuring the PDA Plugin connections on RedHat OpenShift 4.8. As a workaround, update the entando-plugin Role by adding the missing rules below.
```

Comment:
isn't this supposed to have the bit about circumventing the connection error? otherwise everything else looked fine.

Reply:
i reworded it... "circumventing" (my word choice) sounded not quite right; it implies dodging something that's an inevitability. "workaround" is standard language for a quick and dirty - and temporary - fix. but explicitly calling out that something will error gives off an unfinished, amateur vibe, instead of finessing the inferences like in the revision.

Reply:
I agree that the mention of an error is not necessarily a good thing but sometimes I think the meaning starts to get a bit vague. In this case, it is understandable but not crystal clear.

Reply:
good point